batch_job/src/amazon_batch_cleanup_script.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/sh
+#
+# File: amazon_batch_cleanup_script.sh
+# Author: Kyle Sweeney
+#
+# Created on Jul 17, 2017
+#
+
+configFile=$1
+
+ec2_vpc=$(python -c "import json; print json.load(open('$configFile','r'))['vpc'];")
```

Comment:
That's a neat idea, it didn't occur to me to do it that way.
Does this work in both python 2 and python 3?

Reply:
Since it depends in such a way in python, it is a shell script only in name. Why don't we make it a proper python script?